### PR TITLE
Add VS Code Build/Debug docs for Linux.

### PR DESCRIPTION
**Pull request by johnkord**
This is a rough draft of translating @anakrish's PR for adding Windows docs for how to build/debug enclaves with VSCode over to Linux.

Please review these changes here in pretty-printed format: https://github.com/openenclave/openenclave/blob/linux-vscode-debugging/docs/GettingStartedDocs/Linux_vscode.md

(Note: this was reopened now directed at the master branch after @anakrish's PR just got merged today. There were no changes made since the last PR)

### CODE REVIEW

**Comment by dthaler** [comment]
@HernanGatta please review with respect to the other VS Code extension

File: docs/GettingStartedDocs/Linux_vscode.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+2. Run VS Code from any modern Windows system and use the `Remote SSH` extension in VS Code to connect to a Linux system you wish to develop on.
+  a. For more information on how to use the `Remote SSH` extension, please follow this blog post here: [https://code.visualstudio.com/blogs/2019/07/25/remote-ssh](https://code.visualstudio.com/blogs/2019/07/25/remote-ssh)
+
+## Install VS Code
+
+The latest version of Visual Studio Code can be installed from [https://code.visualstudio.com/](https://code.visualstudio.com/)
+
+## Install VS Code Extensions
+
+Install the following VS Code extensions. If you are using the Remote SSH, ensure they are installed remotely too by clicking  Click on an image to navigate to the Visual Studio Code Marketplace page for the extension.
```
Comment (anakrish) [inline docs/GettingStartedDocs/Linux_vscode.md]: Some words seem to have been cut off "ensure they are installed remotely too by clicking Click..."

Comment (johnkord) [reply docs/GettingStartedDocs/Linux_vscode.md]: Thanks Anand! I wonder how that happened here. I don't remember writing this. Maybe I accidentally pasted something...

**Comment by johnkord** [comment]
Thanks for the reviews all! :)
bors r+

**Pull request merged**